src/schema/v2/artwork/index.ts

```diff
@@ -448,6 +449,19 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLBoolean,
         resolve: ({ display_price_range }) => display_price_range,
       },
+      downloadableImageUrl: {
+        type: GraphQLString,
+        resolve: ({ id, images }) => {
+          if (!images || !images.length) return null
+
+          const defaultImage =
+            images.find((image) => image.is_default) || images[0]
+
+          if (!defaultImage) return null
+
+          return `${config.GRAVITY_API_BASE}/artwork/${id}/image/${defaultImage.id}/original.jpg`
```

Comment:
Note that you can avoid specifying the image id if you only need the default:

`/artwork/:id/default_image/original.jpg`